solana/pyth2wormhole/program/src/config.rs

```diff
@@ -154,7 +154,7 @@ impl From<Pyth2WormholeConfigV2> for Pyth2WormholeConfigV3 {
             wh_prog,
             pyth_owner,
             max_batch_size,
-            is_active,
+            is_active: _,
```

Comment:
I think it's wrong that we forced is_active to `true` with this, but admittedly this is being used rarely enough that it shouldn't matter

Reply:
yeah. it does not matter now anymore.